added key fingerprint in .ssh documentation (#31)
UMass-RC/unity-docs: docs/connecting/ssh.md

!!! note

>     Your username should be your email with all at's `@` and dot's `.` replaced with underscores `_`. View your username [here](https://unity.rc.umass.edu/panel/account.php)

Reviewer: This is not strictly correct. Often times this is true, but we use EPPN (Edu-person principal name) to create usernames, and sometimes people will have emails that are not the same as their netids. (I know this isn't part of this PR but I just spotted it, so could you update that as well?)

```suggestion
    Your username is in the format `<organization username>_<organization>_edu`. View your username [here](https://unity.rc.umass.edu/panel/account.php)
```

![PuTTY Save](res/putty-save.png)

The server's ssh-ed25519 key fingerprint is:

> ssh-ed25519 255 SHA256:jC7BF7h5/RJo5Svx1v+lufdf+I/ogu5dQV2sUe+y8ek

Reviewer: ```suggestion
`ssh-ed25519 255 SHA256:jC7BF7h5/RJo5Svx1v+lufdf+I/ogu5dQV2sUe+y8ek`
```

Wrap these in tickmarks so they show up as a code block
